pkg/virt-handler/vm_test.go

```diff
@@ -54,9 +54,15 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-handler/isolation"
 	virtlauncher "kubevirt.io/kubevirt/pkg/virt-launcher"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/network"
 	"kubevirt.io/kubevirt/pkg/watchdog"
 )
 
+func StubOutNetworkForTest() {
+	network.SetupPodNetworkPhase1 = func(vm *v1.VirtualMachineInstance) error { return nil }
+	network.SetupPodNetworkPhase2 = func(vm *v1.VirtualMachineInstance, domain *api.Domain) error { return nil }
+}
+
```

Comment:
Could you add a test for the migration? Would it be possible? Or is it totally insane?

Reply:
I don't think this can be effectively tested. There are plenty of migration functional tests though.

Or do you mean old files migration?

Reply:
I mean migration of the old files to the new directory.

Reply:
This code is gone now.